tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/ClientCredentialsTests.NetFwk.cs

```diff
@@ -63,7 +63,15 @@ public async Task WithCertificate_TestAsync(Cloud cloud, TargetFrameworks runOn,
         }
 
         [DataTestMethod]
-        [DataRow(Cloud.Public,  TargetFrameworks.NetCore)]
+        [DataRow(Cloud.Public, TargetFrameworks.NetFx | TargetFrameworks.NetCore)]
+        public async Task WithMtlsCertificate_TestAsync(Cloud cloud, TargetFrameworks runOn, bool useAppIdUri = false)
```

Comment:
Pls don't use this `appIdUri`. It's quite obscure config and integration tests are expensive.